public/app/features/alerting/unified/components/contact-points/ContactPoints.tsx

```diff
@@ -193,24 +193,18 @@ const ContactPointsPageContents = () => {
 interface ContactPointsListProps {
   contactPoints: ContactPointWithMetadata[];
   search?: string | null;
-  disabled?: boolean;
```

Comment:
It seems like this prop is no longer used since we moved the delete logic to the singular contact point component.